src/manage.py

```diff
@@ -468,12 +468,32 @@ def update_element_tags(self):
     self.library.tags_updated = datetime.datetime.now()
     return tag_map
 
+  def update_package_tags(self):
```

Comment:
Yep. There's a variety of 'tasks' throughout the system. The update tasks get triggered by a cron job every 10 minutes.